package.json

```diff
@@ -70,18 +70,18 @@
     "karma-benchmark": "^0.4.0",
     "karma-chrome-launcher": "^0.2.0",
     "karma-firefox-launcher": "^0.1.6",
-    "karma-junit-reporter": "^0.2.2",
+    "karma-junit-reporter": "^0.3.3",
     "karma-safari-launcher": "^0.1.1",
-    "lodash": "~2.4.1",
-    "minifyify": "7.0.0",
+    "lodash": "~3.10.1",
```

Comment:
That was a mistake on my part, I intended to keep to npm's default of using the caret. However, given that there are already different range rules being used, is there a preferred rule going forward?